development/webpack/webpack.config.ts

```diff
@@ -260,6 +260,12 @@ const config = {
     rules: [
       // json
       { test: /\.json$/u, type: 'json' },
+      // treats compressed JSON files loaded via `new URL('./file.json.zst', import.meta.url)` as assets.
+      {
+        test: /\.json.gz$/u,
```

Comment:
I think you should just combine this one with the `test: /\.json$/u,` one rule down.